README.md

```diff
@@ -3,21 +3,19 @@ Code Runner: A tool to help run your assignments
 Code Runner is a web application to run code pertaining to your assignment using user provided configurations.
 
 ## Features
-There are two roles in the application - Professor and Student.
-#### Professor
-- Professor provides course specific configurations for creating suitable docker environment.
-- Course specific configurations include specifying operating system and its version, programming language and compiler.
+There is one role in the application - Student.
 #### Student
```

Comment:
we can get rid of this sub-heading for the same reason mentioned above.

Reply:
Done